test/unit/controller/predicates/is_request_sources_only_whosonfirst.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+const _ = require('lodash');
+const is_request_sources_only_whosonfirst = require('../../../../controller/predicates/is_request_sources_only_whosonfirst');
+
+module.exports.tests = {};
+
+module.exports.tests.interface = (test, common) => {
+  test('valid interface', (t) => {
+    t.ok(_.isFunction(is_request_sources_only_whosonfirst), 'is_request_sources_only_whosonfirst is a function');
+    t.end();
+  });
+};
+
+module.exports.tests.true_conditions = (test, common) => {
+  test('sources only \'whosonfirst\' should return true', (t) => {
+    const req = {
+      clean: {
+        sources: [
+          'whosonfirst'
+        ]
+      }
+    };
+
+    t.ok(is_request_sources_only_whosonfirst(req));
+    t.end();
+
+  });
+
+};
+
+module.exports.tests.false_conditions = (test, common) => {
+  test('undefined req should return false', (t) => {
```

Comment:
This test looks identical to test `sources not \'whosonfirst\' should return false`. Isn't `req` defined here?